### DISCUSSION TITLE
[Mobile] UnderlinedSpan is not displayed on the selected text in the editor

### DISCUSSION THREAD

**Discussion by franciscrispin** [singerdmx/flutter-quill#590] [Mobile]
The UnderlinedSpan does not display for the text that the cursor is on in the flutter quill editor. Refer to the screenshots for examples. Note this issue is not referring to the underline option provided by the toolbar.

For more context about UnderlinedSpan, it seems to be based on the user device settings but the flutter quill editor seems like it does not follow the settings. 

I suspect this may be intentional because the UnderlinedSpan may conflict with the underline toolbar option? If so, is there a workaround where we can include the UnderlinedSpan for the text that the cursor is on if the option is enabled in the user's device?

For more info about the UnderlinedSpan: https://stackoverflow.com/a/52587675 

**Examples**

TextFormField with the text underlined when the cursor is on the text:

![image](https://user-images.githubusercontent.com/35050056/137582257-3d1a0436-a446-4e51-a27a-99824841ab99.png)

Flutter quill editor with the text NOT underlined when the cursor is on the text:

![image](https://user-images.githubusercontent.com/35050056/137582280-22e0971c-80fb-4705-9837-e5a55dfe202a.png)

Screen recording to demonstrate:

https://user-images.githubusercontent.com/35050056/137582359-fda7deb5-a33e-41df-a084-5de26611d169.mp4

**Specs**

- Flutter version: 2.5.2
- Device: Pixel 4 API 30

**Comment by singerdmx** [comment]
Please Submit a PR
